frontend/src/components/object/ObjectFileDetails.vue

```diff
@@ -54,12 +54,7 @@ const showPermissions = async (objectId: string) => {
 };
 
 function onDeletedSuccess() {
-  toast.add({
-    severity: 'success',
-    summary: 'Success',
-    detail: 'File deleted',
-    life: 3000
-  });
+  toast.success('Deleting file', 'File deleted');
```

Comment:
Style: Jarring presentation -> present-progressive and a past participle statement in one toast. Would recommend consulting with Tyler on this if this hasn't been raised.